requirements.txt

```
@@ -7,3 +7,4 @@ pandas==1.5.2
 scikit_learn==1.1.3
 scipy==1.9.3
 xarray==2022.11.0
+pytest
```

Can we fix or bound the `pytest` version?
```suggestion
pytest>=1.0.0
```